packages/selection/demo/stories/SelectionStory.tsx

```diff
@@ -16,86 +17,122 @@ import {
   useSelection
 } from '@zendeskgarden/container-selection';
 
+interface ISelectionElementProps {
+  getElementProps: IUseSelectionReturnValue<any>['getElementProps'];
+  value: string;
+  index: number;
+  direction: IUseSelectionProps<any>['direction'];
+  selectedValue: IUseSelectionProps<any>['selectedValue'];
+}
+
+const SelectionElement = ({
+  getElementProps,
+  value,
+  index,
+  direction,
+  selectedValue
+}: ISelectionElementProps) => {
+  const { ref: selectionRef, ...elementProps } = getElementProps({ value });
+  const focusRef = useRef(null);
```

Comment:
Why is `focusRef` needed here? If it isn't, then the `react-merge-refs` dependency can go away.